# Service to Set Cartesians
float64 x
float64 y
float64 z
float64 qw
float64 qx
float64 qy
float64 qz
---
